openpdf/src/main/java/com/lowagie/text/pdf/PdfDocument.java

```diff
@@ -528,13 +529,19 @@ public boolean add(Element element) throws DocumentException {
                     // if a paragraph has to be kept together, we wrap it in a table object
                     if (paragraph.getKeepTogether()) {
                         carriageReturn();
-                        PdfPTable table = new PdfPTable(1);
-                        table.setWidthPercentage(100f);
-                        PdfPCell cell = new PdfPCell();
-                        cell.addElement(paragraph);
-                        cell.setBorder(Table.NO_BORDER);
-                        cell.setPadding(0);
-                        table.addCell(cell);
+                        Table table = new Table(1);
```

Comment:
I've tried and got an IllegalArgumentException: Element not allowed at this point:
https://github.com/LibrePDF/OpenPDF/blob/master/openpdf/src/main/java/com/lowagie/text/pdf/ColumnText.java#L462

I've been using Table instead of PdfPTable. But when I've changed it to PdfPTable everything worked as expected.
I think it's ok if it's not working with Table, because you can change Table easiely to PdfPTable.